server/weaverbird/backends/sql_translator/types.py

```diff
@@ -20,87 +20,85 @@ class TableMetadataUpdateError(Exception):
 
 class SqlQueryMetadataManager(BaseModel):
     tables_metadata: Dict[str, Dict[str, str]]
```

Comment:
Du coup on ne crée pas les objets Table et Column pour l'instant ?

Reply:
oui est-ce qu'on pourra l'ajouter sur une autre PR plus tard ?

Reply:
Tout à fait, crée bien la carte en lien avec ça

Reply:
https://trello.com/c/9vvc6QF0/1167-snowflake-sql-vqb-as-a-dev-i-implement-the-column-table-object-in-metadata